cmd/confluence/confluence.go

```diff
@@ -675,9 +750,9 @@ func (j *DSConfluence) EnrichItem(ctx *shared.Ctx, item map[string]interface{})
 	user, _ := shared.Dig(version, []string{"by"}, true, false)
 	rich["by"] = user
 	rich["message"], _ = shared.Dig(version, []string{"message"}, false, true)
-	iVersion, _ := version["number"]
+	iVersion := version["number"]
 	rich["version"] = iVersion
-	rich["date"], _ = version["when"]
```

Comment:
removing `, _` part can cause panics. I would revert that paryicular change.

Reply:
@lukaszgryglicki Got it